pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -12100,7 +12101,7 @@ func TestReconcile_verifyResolved_V1Pipeline_Error(t *testing.T) {
 	resolverName := "foobar"
 
 	// Case1: unsigned Pipeline refers to unsigned Task
-	unsignedTask := parse.MustParseV1beta1Task(t, `
+	unsignedV1beta1Task := parse.MustParseV1beta1Task(t, `
```

Comment:
shouldn't this test class be testing the v1 api for remote tasks/pipelines, even if the pipelinerun itself is v1beta1?

Reply:
I do agree, introduced in https://github.com/tektoncd/pipeline/pull/6765/files, I changed to v1Tasks and v1Pipeline in the current PR. just cc @Yongxuanzhang to double check on this.

Reply:
Oh thanks for the fix! This is a mistake for the `TestReconcile_verifyResolved_V1Pipeline_Error`